src/strategies/priority_strategy.rs

```diff
@@ -137,11 +137,14 @@ impl<M: Middleware + 'static> Strategy<Event, Action> for UniswapXPriorityFill<M
     }
 
     // Process incoming events, seeing if we can arb new orders, and updating the internal state on new blocks.
-    async fn process_event(&mut self, event: Event) -> Option<Action> {
+    async fn process_event(&mut self, event: Event) -> Vec<Action> {
```

Comment:
why changing the return type here?

Reply:
This was due to an artemis-core package update.. for some reason the compiler auto-updated it to a different version. I've reverted the cargo.toml + lock changes in addition to this.